src/inference/dev_api/openvino/runtime/internal_properties.hpp

```diff
@@ -61,6 +101,42 @@ static constexpr Property<std::string, PropertyMutability::WO> config_device_id{
 static constexpr Property<ov::threading::IStreamsExecutor::ThreadBindingType, PropertyMutability::RW> cpu_bind_thread{
     "CPU_BIND_THREAD"};
 
+/**
+ * @brief Number of streams in Performance-core(big core)
+ * @ingroup ov_dev_api_plugin_api
+ */
+static constexpr Property<size_t, PropertyMutability::RW> big_core_streams{"BIG_CORE_STREAMS"};
+
+/**
+ * @brief Number of streams in Efficient-core(small core) on hybrid cores machine
+ * @ingroup ov_dev_api_plugin_api
+ */
+static constexpr Property<size_t, PropertyMutability::RW> small_core_streams{"SMALL_CORE_STREAMS"};
+
+/**
+ * @brief Number of threads per stream in big cores
+ * @ingroup ov_dev_api_plugin_api
+ */
+static constexpr Property<size_t, PropertyMutability::RW> threads_per_stream_big{"THREADS_PER_STREAM_BIG"};
+
+/**
+ * @brief Number of threads per stream in small cores on hybrid cores machine
+ * @ingroup ov_dev_api_plugin_api
+ */
+static constexpr Property<size_t, PropertyMutability::RW> threads_per_stream_small{"THREADS_PER_STREAM_SMALL"};
+
+/**
+ * @brief Small core start offset when binding cpu cores
+ * @ingroup ov_dev_api_plugin_api
+ */
+static constexpr Property<size_t, PropertyMutability::RW> small_core_offset{"SMALL_CORE_OFFSET"};
+
+/**
+ * @brief Enable hyper thread
+ * @ingroup ov_dev_api_plugin_api
+ */
+static constexpr Property<bool, PropertyMutability::RW> enable_hyper_thread{"ENABLE_HYPER_THREAD"};
```

Comment:
we already have public property `enable_hyper_threading`, can we use it?

Reply:
"ENABLE_HYPER_THREAD" is used by legacy config name , new api has changed it to be "ENABLE_HYPER_THREADING".
yes, we don't need it.